lib/hammer_cli_katello/filter.rb

```diff
@@ -53,6 +53,15 @@ class CreateCommand < HammerCLIKatello::CreateCommand
       success_message _("Filter created")
       failure_message _("Could not create the filter")
 
+      validate_options do
+        organization_options = [:option_organization_id, :option_organization_name, \
+                                :option_organization_label]
+
+        if option(:option_repository_names).exist? || option(:option_content_view_name).exist?
+          any(*organization_options).required
```

Comment:
So @zjhuntin I think we agree to remove this validation of organization options, unless you have objections.